lib/database.ts

```diff
@@ -189,6 +189,11 @@ export class Database {
     return this.getConnector()._client;
   }
 
+  /* Get the database pool if existent. */
+  getPool?() {
```

Comment:
Can we drop the `?` here?

```suggestion
  getPool() {
```